lib/hammer_cli_csv.rb

```diff
@@ -6,8 +6,9 @@ module HammerCLICsv
   require 'hammer_cli_foreman_tasks'
 
   require 'hammer_cli_csv/i18n'
-  require 'hammer_cli_csv/base'
+  require 'hammer_cli_csv/utils/config'
```

Comment:
created utils/config to set up reading config file/settings. used in base, import, and export